src/core/inductive.ml

```diff
@@ -0,0 +1,63 @@
+(** Builtin configuration for induction. *)
```

Comment:
This is a comment for the type `config` not for the whole file. Thanks to move this comment below and add one for the file.